test/petab/test_amici_prediction.py

```diff
@@ -296,7 +299,55 @@ def test_petab_prediction():
     # create prediction via PAteb
     prediction = importer.create_prediction()
 
-    # run test
+    # ===== run test for prediction ===========================================
     p = prediction(np.array(petab_problem.x_nominal_free_scaled),
                    sensi_orders=(0, 1))
     check_outputs(p, out=(0, 1), n_cond=1, n_timepoints=10, n_obs=1, n_par=2)
+    # check outputs for simulation and measurement dataframes
+    importer.petab_prediction_to_measurement_df(p, prediction)
+    importer.petab_prediction_to_simulation_df(p, prediction)
+
+    # ===== run test for ensemble prediction ==================================
+    # read a set of ensemble vectors from the csv
+    ensemble_file = os.path.join(
+        os.path.dirname(__file__), '..', '..', 'doc', 'example', model_name,
+        'parameter_ensemble.tsv')
+    ensemble = pypesto.collections.read_from_csv(
+        ensemble_file, lower_bound=petab_problem.get_lb(),
+        upper_bound=petab_problem.get_ub())
+    isinstance(ensemble, Collection)
+
+    # check summary creation and identifiability analysis
+    summary = ensemble.compute_summary(percentiles_list=[10, 25, 75, 90])
+    assert isinstance(summary, dict)
+    assert set(summary.keys()) == {'mean', 'std', 'median', 'percentile 10',
+                                   'percentile 25', 'percentile 75',
+                                   'percentile 90'}
+
+    parameter_identifiability = ensemble.check_identifiability()
+    assert isinstance(parameter_identifiability, pd.DataFrame)
+
+    # perform a prediction for the ensemble
+    ensemble_prediction = ensemble.predict(predictor=prediction)
```

Comment:
`predictor=prediction` seems wrong? Maybe some things could be renamed

Reply:
yeah, agreed, the naming is inconsistent. Will change :)